packages/node-experimental/README.md

```diff
@@ -50,10 +50,26 @@ Currently, this SDK:
 * Will capture errors (same as @sentry/node)
 * Auto-instrument for performance - see below for which performance integrations are available.
 
+### Manual Instrumentation
+
 **Manual instrumentation is not supported!**
 This is because the current Sentry-Performance-APIs like `Sentry.startTransaction()` are not compatible with the OpenTelemetry tracing model.
 We may add manual tracing capabilities in a later version.
 
+### ESM Support
+
+Due to the way OpenTelemetry handles instrumentation, this only works out of the box for CommonJS (`require`) applications.
+
+
+There is experimental support for running OpenTelemetry with ESM (`"type": "module"`):
+
+```bash
+node --experimental-loader=@opentelemetry/instrumentation/hook.mjs ./app.js
```

Comment:
😭

Reply:
yes, so sad 😬 and even that doesn't work with all integrations, e.g. it fails with fastify. 😢